content/en/docs/setup/install/istioctl/index.md

```diff
@@ -335,12 +335,12 @@ The `IstioOperator` API defines components as shown in the table below:
 | ------------|
 `base` |
 `pilot` |
-`proxy` |
 `telemetry` |
 `policy` |
 `ingressGateways` |
 `egressGateways` |
 `cni` |
+`istiodRemote` |
```

Comment:
I think you can also remove `telemetry` and `policy`.
Ref: https://preliminary.istio.io/latest/docs/reference/config/istio.operator.v1alpha1/#IstioComponentSetSpec

Reply:
done.